docs/02-membership.md

```diff
@@ -134,7 +134,7 @@ The membership is a set of people working within the eligible projects who have
 | [Ahmad Bitar](https://github.com/smartprogrammer93 ) | 0.5 | Nethermind | |
 | [Alexey Osipov](https://github.com/flcl42) | 1 | Nethermind | [NethermindEth contributions](https://github.com/flcl42?org=NethermindEth)<br>[NethermindEth/nethermind PR's](https://github.com/NethermindEth/nethermind/pulls/flcl42) |
 | [Ayman Bouchareb](https://github.com/Demuirgos) | 1 | Nethermind | |
-| [Ben Adams](https://github.com/benaadams) | 0.5 | Nethermind | |
+| [Ben Adams](https://github.com/benaadams) | 0.5 | Nethermind | [NethermindEth contributions](https://github.com/NethermindEth/nethermind/pulls?q=author%3Abenaadams)|
```

Comment:
can you include the link in this format?

NethermindEth/nethermind

Reply:
Like that?

Reply:
yep, thanks!